packages/dataviz/src/components/LineChart/LineChart.stories.tsx

```diff
@@ -63,101 +63,125 @@ export const FullyCustomisedLineChart = {
 			{
 				key: 'trustScore',
 				color: '#1667DF',
-				dashed: true,
 				tooltipLabel: 'Trust Score™',
 				legendLabel: 'Talend Trust Score™',
 				axis: 'right',
-				status: 'active',
+				status: 'light',
 			},
 			{
 				key: 'validity',
-				color: '#B045E5',
+				color: '#D2AD15',
```

Comment:
Have we tried using the tokens here? We do export then in TSX as well.
If it doesn't work, there's this: https://github.com/Talend/ui/pull/3552

Reply:
You're right, in the real world, we use tokens 😄
I will see if i can update them using the DS-Tokens 👍🏻